x-pack/plugins/enterprise_search/public/applications/app_search/components/layout/kibana_header_actions.tsx

```diff
@@ -0,0 +1,29 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React from 'react';
+
+import { EuiButtonEmpty, EuiText, EuiFlexGroup, EuiFlexItem, EuiHeaderLinks } from '@elastic/eui';
+import { i18n } from '@kbn/i18n';
+
+export const KibanaHeaderActions: React.FC = () => {
+  return (
+    <EuiHeaderLinks>
+      <EuiFlexGroup gutterSize="s">
+        <EuiFlexItem>
+          <EuiButtonEmpty iconType="beaker">
+            <EuiText size="s">
+              {i18n.translate('xpack.enterpriseSearch.appSearch.engine.queryTesterButtonLabel', {
+                defaultMessage: 'Query tester',
+              })}
+            </EuiText>
+          </EuiButtonEmpty>
```

Comment:
[Optional, feel free to leave for a future PR or start a `query_tester` component folder now] If we're keeping `KibanaHeaderActions` as a future-proof component to potentially expand with more actions at some point, I'd suggest not hard-coding in specific buttons but instead importing them from their conceptual parents, e.g.:

```js

import { QueryTesterButton } from '../query_tester';

//..
<EuiFlexItem>
  <QueryTesterButton />
</EuiFlexItem>
// ...
```

Reply:
I'll consider this for a future PR. I don't know if it's worth it to encapsulate the button in a separate component. I'll see what it looks like when it starts coming together!

Reply:
I will do this in the next PR